types/angular-translate/index.d.ts

```diff
@@ -56,6 +58,21 @@ declare module 'angular' {
             instant(translationId: string, interpolateParams?: any, interpolationId?: string, forceLanguage?: string, sanitizeStrategy?: string): string;
             instant(translationId: string[], interpolateParams?: any, interpolationId?: string, forceLanguage?: string, sanitizeStrategy?: string): { [key: string]: string };
             isPostCompilingEnabled(): boolean;
+            /**
+             * @ngdoc function
+             * @name pascalprecht.translate.$translate#negotiateLocale
+             * @methodOf pascalprecht.translate.$translate
+             *
+             * @description
+             * Returns a language key based on available languages and language aliases. If a
+             * language key cannot be resolved, returns undefined.
+             *
+             * If no or a falsy key is given, returns undefined.
+             *
+             * @param {string} [key] Language key
+             * @return {string|undefined} Language key or undefined if no language key is found.
```

Comment:
The types in the JSDoc in `d.ts` are overengineering. I propose to simplify.
```
             * @param key Language key
             * @return Language key or undefined if no language key is found.
```

Reply:
That's just a copy-paste from the original code. But agree, it's repetition :)

Reply:
Removed

Reply:
@yrtimiD you forgot about brackets (`[]`).

Reply:
@yrtimiD you forgot about brackets (`[]`).